bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/help/ProductInformationHandler.java

```diff
@@ -0,0 +1,102 @@
+package com.espressif.idf.ui.help;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.core.runtime.Platform;
+import org.ini4j.InvalidFileFormatException;
+import org.ini4j.Wini;
+
+import com.espressif.idf.core.IDFCorePlugin;
+import com.espressif.idf.core.IDFEnvironmentVariables;
+import com.espressif.idf.core.ProcessBuilderFactory;
+import com.espressif.idf.core.logging.Logger;
+import com.espressif.idf.core.util.IDFUtil;
+import com.espressif.idf.ui.update.ListInstalledToolsHandler;
+import com.espressif.idf.ui.update.Messages;
+
+
+public class ProductInformationHandler extends ListInstalledToolsHandler
+{
+	@Override
+	public Object execute(ExecutionEvent event) throws ExecutionException
+	{
+		super.execute(event);
+		console.println(Messages.JavaRuntimeVersionMsg + System.getProperty("java.runtime.version"));
+		console.println(Messages.OperationSystemMsg + System.getProperty("os.name").toLowerCase());
+		console.println(Messages.EclipseCDTMsg + Platform.getBundle("org.eclipse.cdt").getVersion().toString());
+		console.println(Messages.PythonIdfEnvMsg + getPythonExeVersion(IDFUtil.getIDFPythonEnvPath()));
+		console.println(Messages.PythonPathMsg + getPythonExeVersion("python"));
+		showEclipseVersion();
+		showEspIdfVersion();
+		return null;
```

Comment:
How about re-ordering these messages

Operating System:
Java Version:
Eclipse Version:
Eclipse CDT Version:
IDF Eclipse Plugin version:
ESP-IDF version:
Python versions

Tools information:


Eclipse CDT Build environment variables
IDF_PATH
IDF_PYTHON_ENV_PATH
PATH

Reply:
https://github.com/espressif/idf-eclipse-plugin/blob/cdaf7a1e2089498ec9e7f874cb037a9981f8ee60/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/help/ProductInformationHandler.java#L32-L46

@kolipakakondal messages were reordered, but Tools information is going first. The reason for that, 
super.execute(event); activates and clears console output. 

So the output is next:
Tools information:

Eclipse CDT Build environment variables: (Read all environment variables defined and print)
IDF_PATH
IDF_PYTHON_ENV_PATH
PATH

Operating System:
Java Version:
Eclipse Version:
Eclipse CDT Version:
IDF Eclipse Plugin version:
ESP-IDF version:
Python versions